src/RA_Defs.h

```diff
@@ -87,26 +87,24 @@
 
 
 
-
-//	RA-Only
-
-
-//	RA-Only
-#define RAPIDJSON_HAS_STDSTRING 1
-#pragma warning(push, 1)
-// This is not needed the most recent version
-#define _SILENCE_ALL_CXX17_DEPRECATION_WARNINGS
 //	RA-Only
+#pragma warning(push)
+// Seems version 1.1.0 still uses deprecated stuff.
+#pragma warning(disable : 4996) // STL4015 deprecation: uses iterator as a base class instead of usings from iterator_traits.
+// NOMINMAX doesn't seem to have an effect on rapidjson
```

Comment:
It does, but only for those files that include "RA_Defs.h" before any other windows header files. I had to add `NOMINMAX` to the following files to eliminate all of the warnings:
* RA_Dlg_Login.h
* RA_Dlg_RichPresence.h
* RA_Dlg_RomChecksum.h
* RA_ImageFactory.h
* RA_Errors.h

I just assumed you weren't seeing the warnings, because I also had to add "#include <memory>" to RA_AchievementOverlay.cpp, RA_AchievementSet.cpp and RA_GameData.h to even get this PR to compile. Whatever you're doing to define "std::unique_ptr" and "std::make_unique" isn't happening on my machine.